learning_resources_search/api.py

```diff
@@ -21,11 +21,11 @@
     DEPARTMENT_QUERY_FIELDS,
     LEARNING_RESOURCE,
     LEARNING_RESOURCE_QUERY_FIELDS,
-    LEARNING_RESOURCE_SEARCH_FILTERS,
     LEARNING_RESOURCE_TYPES,
     RESOURCEFILE_QUERY_FIELDS,
     RUN_INSTRUCTORS_QUERY_FIELDS,
     RUNS_QUERY_FIELDS,
+    SEARCH_FILTERS,
```

Comment:
Renamed because LEARNING_RESOURCE_SEARCH_FILTERS contains both Learning Resource and Content File search entries